scipy/ndimage/tests/test_morphology.py

```diff
@@ -1126,13 +1116,11 @@ def test_binary_erosion29(self, xp):
                                      border_value=1, iterations=3)
         assert_array_almost_equal(out, expected)
 
-    @skip_xp_backends(
-        np_only=True, reason='inplace out= arguments are numpy-specific'
-    )
+    @skip_xp_backends(np_only=True, exceptions=["numpy"],
+                      reason='inplace out= arguments are numpy-specific')
```

Comment:
What does this mean: np_only=True, exceptions="numpy"?

Reply:
Oops, looks like a typo

Reply:
#22148

Reply:
it was meant to be exceptions=["cupy"]. I wanted to xfail cupy and skip the rest.